gradle/libs.versions.toml

```diff
@@ -38,8 +38,8 @@ androidx-webkit = '1.12.1'
 androidx-work = '2.9.1'
 apache-commons-text = '1.13.0'
 apache-http-client-android = '4.3.5.1'
+automattic-about = '1.5.1'
 assertj = '3.27.3'
-automattic-about = '1.4.0'
```

Comment:
While the previous list appeared to be alphabetized, this change relocates `automattic-about` to no longer be in alphabetical order. Obviously not a significant issue nor one we need to address now, merely noting it.